app/templates/index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/app/static/index.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="background">
+        <!--Background-->
+        <svg width="1198" height="722" viewBox="0 0 1398 722" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
- [ optional] instead of this, you can download the main file containing all this svgs by exporting it and saving it. When you have it, you can set it from CSS as background image instead of using it directly